src/Umbraco.Web.UI/umbraco/config/lang/en.xml

```diff
@@ -170,6 +170,7 @@
     <key alias="updateDate">Last edited</key>
     <key alias="updateDateDesc" version="7.0">Date/time this document was edited</key>
     <key alias="uploadClear">Remove file(s)</key>
+    <key alias="url">URL</key>

```

Comment:
Hmmm yeah - we might need that at some point - I've seen that for other sections where things are starting to get a little messy. Oh well.. We'll see.. may need to add it in again at some point :)